player/stream-test/js/script.js

```diff
@@ -82,21 +96,42 @@ function checkIsUrlValid(url, urlPurpose) {
   return true;
 }
 
-function check404(url) {
-  return new Promise(function (resolve, reject) {
-    var xhr = new XMLHttpRequest();
-    xhr.open('GET', url, true);
-    xhr.onreadystatechange = function () {
-      if (xhr.status !== 200) {
-        xhr.onreadystatechange = null;
-        reject(xhr.status);
-      } else {
-        xhr.onreadystatechange = null;
-        resolve(200);
-      }
-    };
-    xhr.send('Content-type', 'application/x-www-form-urlencoded');
-  });
+function getParamsQueryString() {
+  var key = false, res = {}, itm = null;
```

Comment:
Don't shorten names, this just makes code unreadable. 
Also, even though it is a dynamic language, don't change types of variables. You could also check for `null` instead of `false.